ui/packages/consul-ui/mock-api/v1/discovery-chain/virtual-admin-6

```diff
@@ -0,0 +1,196 @@
+      },
+      "splitter:login.default.default": {
+        "Type": "splitter",
+        "Name": "login.default.default",
+        "Splits": [
+          {
+            "Weight": 50,
+            "NextNode": "resolver:web.default.default.dc-gcp"
+          },
+          {
+            "Weight": 25,
+            "NextNode": "resolver:web.default.default.dc-aws"
+          },
+          {
+            "Weight": 25,
+            "NextNode": "resolver:db.default.partition.dc-gcp"
+          }
+        ]
+      }
+    },
+    "Targets": {
+      "virtual-admin-6.default.default.dc-gcp": {
+        "ID": "virtual-admin-6.default.default.dc-gcp",
+        "Service": "virtual-admin-6",
+        "Partition": "default",
+        "Namespace": "default",
+        "Datacenter": "dc-gcp",
+        "MeshGateway": {},
+        "Subset": {},
+        "SNI": "virtual-admin-6.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul",
+        "Name": "virtual-admin-6.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul"
+      },
+      "web.default.default.dc-aws": {
+        "ID": "web.default.default.dc-aws",
+        "Service": "web",
+        "Partition": "default",
+        "Namespace": "default",
+        "Datacenter": "dc-aws",
+        "MeshGateway": {
+          "Mode": "local"
+        },
+        "Subset": {},
+        "SNI": "web.default.default.dc-aws.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul",
+        "Name": "web.default.default.dc-aws.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul"
+      },
+      "web.default.default.dc-gcp": {
+        "ID": "web.default.default.dc-gcp",
+        "Service": "web",
+        "Partition": "default",
+        "Namespace": "default",
+        "Datacenter": "dc-gcp",
+        "MeshGateway": {},
+        "Subset": {},
+        "SNI": "web.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul",
+        "Name": "web.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul"
+      },
+      "db.default.partition.dc-gcp": {
+        "ID": "db.default.partition.dc-gcp",
+        "Service": "db",
+        "Partition": "partition",
+        "Namespace": "default",
+        "Datacenter": "dc-gcp",
+        "MeshGateway": {},
+        "Subset": {},
+        "SNI": "db.default.partition.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul",
+        "Name": "db.default.partition.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul"
+      },
+      "db.default.default.dc-gcp": {
+        "ID": "db.default.default.dc-gcp",
+        "Service": "db",
+        "Partition": "default",
+        "Namespace": "default",
+        "Datacenter": "dc-gcp",
+        "MeshGateway": {},
+        "Subset": {},
+        "SNI": "db.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul",
+        "Name": "db.default.default.dc-gcp.internal.ede9daf4-b641-3954-5a52-15f4fd666f6e.consul"
+      }
+    }
+  }
+}
```

Comment:
These are all fixtures from a real Consul install/usecase, well filename numbers 1-5 are, number 6 I added with a partition usecase.

For filenames 1 - 5 I prettified and added the relevant new partition fields, for number 6 I copied number 1 and then added the extra partition usecase to try with.